recipes/libpqxx/all/patches/fix-apple-clang-compilation-7.4.0.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/include/pqxx/connection.hxx b/include/pqxx/connection.hxx
+index b44daa18..21dc1fd3 100644
+--- a/include/pqxx/connection.hxx
++++ b/include/pqxx/connection.hxx
+@@ -33,6 +33,7 @@
+ #include "pqxx/errorhandler.hxx"
+ #include "pqxx/except.hxx"
+ #include "pqxx/prepared_statement.hxx"
++#include "pqxx/separated_list.hxx"
```

Comment:
Could you also send a PR in the upstream, please?

Reply:
Yes. I'm going to check the results of the CI build before PR.

Reply:
This was fixed before PR: jtv/libpqxx@cbd2b35ef5bce8a1fc8bfd1efd804d965976ce23